src/main/java/org/opentripplanner/graph_builder/module/osm/UKWayPropertySetSource.java

```diff
@@ -0,0 +1,80 @@
+package org.opentripplanner.graph_builder.module.osm;
+
+import org.opentripplanner.routing.edgetype.StreetTraversalPermission;
+
+/**
+ * OSM way properties for UK roads. 
+ * The main differences compared to the default property set are:
+ * 1. In the UK there is no real distinction between trunk highways and primary highways, other than the
+ * body responsible for them. Most highway=trunk and highway=trunk_link will allow traversal by all modes.
+ * 2. Speeds have been set to reflect average free flow road speeds provided by UK DfT. In particular
+ * note that a distinction is made between tertiary and unclassified/residential roads. The default has these
+ * the same (25mph) but in the UK tertiary roads are considered by OSM tagging guidelines to be busy unclassified
+ * through roads wide enough to allow two cars to pass safely. The free flow speeds are therefore higher. 
+ * These changes result in more realistic driving routes.
+ * https://www.gov.uk/government/statistical-data-sets/vehicle-speed-compliance-statistics-data-tables-spe
+ * https://wiki.openstreetmap.org/wiki/United_Kingdom_Tagging_Guidelines
+ *   
+ * @author marcusyoung
+ * @see WayPropertySetSource
+ * @see DefaultWayPropertySetSource
+ */
+public class UKWayPropertySetSource implements WayPropertySetSource {
+
+	@Override
+	public void populateProperties(WayPropertySet props) {
+        // Replace existing matching properties as the logic is that the first statement registered takes precedence over later statements
+        props.setProperties("highway=trunk_link", StreetTraversalPermission.ALL, 2.06,
```

Comment:
Indentation seems to be a bit random in this file. Preferably use spaces instead of tabs.

Reply:
@optionsome  I've tidied the identation up.